src/Application/Migrations/06_tickets.sql

```diff
@@ -56,11 +56,18 @@ CREATE OR REPLACE FUNCTION update_ticket_progress()
   RETURNS trigger AS
 $BODY$
   BEGIN
-    UPDATE tbl_ticket SET progress = ticket_progress(NEW.id) WHERE id = NEW.id;
-    IF (NEW.parent_id IS NOT NULL) THEN
-      UPDATE tbl_ticket SET progress = ticket_progress(NEW.parent_id) WHERE id = NEW.parent_id;
+    IF TG_OP = 'DELETE' THEN
+      IF (OLD.parent_id IS NOT NULL) THEN
+        UPDATE tbl_ticket SET progress = ticket_progress(OLD.parent_id) WHERE id = OLD.parent_id;
```

Comment:
Technically you could do that query unconditionally since there should be no rows with id = NULL, but I guess it doesn't matter much.

Reply:
Yes there are no rows with id = NULL but there are rows with parent_id = NULL, i.e. when meta tickets are deleted. Or do you mean that executing ticket_progress(NULL) wouldn't hurt?

Reply:
But parent_id is never part of a WHERE clause. I thought ticket_progress(OLD.parent_id) would never be called, since there are no rows to apply it to. Not sure in which order Postgresql is processing things.

Reply:
But I'm not insisting on the removal of the conditionals. Do as you like.